ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__rosa-sts-test-operator-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: rosa-sts-test-operator
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "operator-install", "failure_type": "all", "classification": "Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "operator-uninstall", "failure_type": "all", "classification": "Deprovisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster (ROSA)", "jira_project": "LPTOCPCI"},
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster (AWS)", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OPENSHIFT_VERSION: "4.14"
```

Comment:
I don't believe there is a rosa 4.14 is available in staging
https://gitlab.cee.redhat.com/service/clusterimagesets/-/blob/master/config/stage.yaml
Context here: https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1689296989307869?thread_ts=1689279329.992419&cid=C04PK4QPSR1

I believe this might explain the error you are seeing here: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43065/rehearse-43065-periodic-ci-rhpit-interop-tests-main-rosa-sts-test-operator-ocp4.14-lp-interop-rosa-sts-test-operator/1699758012970831872#1:build-log.txt%3A30

Switching to 4.13 might resolve this. We can't test rosa-sts-hypershift in staging against 4.14 so I'm thinking we are seeing the same thing here

Reply:
Super helpful, thanks